File: CountConsonants.cs
// Count Consonants
// - Given a string `s`, count all consonant letters.

using System;
using System.Collections.Generic;
using System.Linq;

class Solution
{
    public int CountConsonants(string s)
    {
        string vowels = "aeiouAEIOU";
        return s.Count(c => Char.IsLetter(c) && !vowels.Contains(c));
    }
}

File: CountConsonants.py
# Count Consonants
# - Given a string `s`, count all consonant letters.
# - Constraints:
#   - 1 <= len(s) <= 10^4

class Solution:
    def countConsonants(self, s: str) -> int:
        return sum(1 for c in s if c.isalpha() and c.lower() not in "aeiou")
